bitnami/victoriametrics/values.yaml

```diff
@@ -0,0 +1,3662 @@
+    ##
+    create: true
+    ## @param vmalert.serviceAccount.name The name of the ServiceAccount to use.
+    ## If not set and create is true, a name is generated using the common.names.fullname template
+    ##
+    name: ""
+    ## @param vmalert.serviceAccount.annotations Additional Service Account annotations (evaluated as a template)
+    ##
+    annotations: {}
+    ## @param vmalert.serviceAccount.automountServiceAccountToken Automount service account token for the server service account
+    ##
+    automountServiceAccountToken: false
+  ## @section VictoriaMetrics Alert Traffic Exposure Parameters
+  ##
+  service:
+    ## @param vmalert.service.type VictoriaMetrics Alert service type
+    ##
+    type: ClusterIP
+    ## @param vmalert.service.ports.http VictoriaMetrics Alert service http port
+    ##
+    ports:
+      http: 8429
+    ## Node ports to expose
+    ## @param vmalert.service.nodePorts.http Node port for HTTP
+    ## NOTE: choose port between <30000-32767>
+    ##
+    nodePorts:
+      http: ""
+    ## @param vmalert.service.clusterIP VictoriaMetrics Alert service Cluster IP
+    ## e.g.:
+    ## clusterIP: None
+    ##
+    clusterIP: ""
+    ## @param vmalert.service.loadBalancerIP VictoriaMetrics Alert service Load Balancer IP
+    ## ref: https://kubernetes.io/docs/concepts/services-networking/service/#type-loadbalancer
+    ##
+    loadBalancerIP: ""
+    ## @param vmalert.service.loadBalancerSourceRanges VictoriaMetrics Alert service Load Balancer sources
+    ## ref: https://kubernetes.io/docs/tasks/access-application-cluster/configure-cloud-provider-firewall/#restrict-access-for-loadbalancer-service
+    ## e.g:
+    ## loadBalancerSourceRanges:
+    ##   - 10.10.10.0/24
+    ##
+    loadBalancerSourceRanges: []
+    ## @param vmalert.service.externalTrafficPolicy VictoriaMetrics Alert service external traffic policy
+    ## ref http://kubernetes.io/docs/tasks/access-application-cluster/create-external-load-balancer/#preserving-the-client-source-ip
+    ##
+    externalTrafficPolicy: Cluster
+    ## @param vmalert.service.annotations [object] Additional custom annotations for VictoriaMetrics Alert service
+    ##
+    annotations: {}
+    ## @param vmalert.service.extraPorts Extra ports to expose in VictoriaMetrics Alert service (normally used with the `sidecars` value)
+    ##
+    extraPorts: []
+    ## @param vmalert.service.sessionAffinity Control where client requests go, to the same pod or round-robin
+    ## Values: ClientIP or None
+    ## ref: https://kubernetes.io/docs/concepts/services-networking/service/
+    ##
+    sessionAffinity: None
+    ## @param vmalert.service.sessionAffinityConfig Additional settings for the sessionAffinity
+    ## sessionAffinityConfig:
+    ##   clientIP:
+    ##     timeoutSeconds: 300
+    ##
+    sessionAffinityConfig: {}
+
+  ## @section VictoriaMetrics Alert Metrics Parameters
+  ##
+  ## Prometheus metrics
+  ##
+  metrics:
```

Comment:
Same comment I added at the README, is it worth this section on a solution that's an alternative to Prometheus?

Reply:
If it has a metrics endpoint, I don't see a reason to not scrape it